src/GovUk.Education.ExploreEducationStatistics.Public.Data.Processor/Program.cs

```diff
@@ -0,0 +1,7 @@
+using Microsoft.Extensions.Hosting;
+
+var host = new HostBuilder()
+    .ConfigureFunctionsWorkerDefaults()
```

Comment:
Since we're upgrading to use the isolated process model, I wonder if we should also use the [ASP.NET Core integration](https://learn.microsoft.com/en-us/azure/azure-functions/dotnet-isolated-process-guide?tabs=windows#aspnet-core-integration)? 

We would just need to change this line to `.ConfigureFunctionsWebApplication()`, and include the package reference for `Microsoft.Azure.Functions.Worker.Extensions.Http.AspNetCore`

It then means we can use types from ASP.NET Core like `HttpRequest`, `HttpResponse` and `IActionResult` which would make it more consistent with our other projects.

Reply:
Nice idea! I've added those in now and its much more ergonomic to use the standard .NET Core APIs!